app/main.py

```diff
@@ -2,0 +1,54 @@
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
Similar to the `SoftwareEngineer` class, `skills` is defined as a class attribute here, which will be shared across all instances of `FrontendDeveloper`. Consider moving it to the `__init__` method.